### ISSUE TITLE
So the generated pfm files were not actually used in this pipeline?

### ISSUE THREAD

**Issue by xqyd** [chaytonmin/DeepMVS#6]
I've tested the code and managed to generate multiple pfm files. However, after running openmvs, I realized that these generated pfm files were not adopted at all. The final results still depend on the estimated dmp files by openmvs's DensifyPointCloud.exe, so the generated result have nothing to do with the pfm files?

Please confirm this. Thanks!

**Comment by chaytonmin** [comment]
> I've tested the code and managed to generate multiple pfm files. However, after running openmvs, I realized that these generated pfm files were not adopted at all. The final results still depend on the estimated dmp files by openmvs's DensifyPointCloud.exe, so the generated result have nothing to do with the pfm files?
> 
> Please confirm this. Thanks!

Yes, you are right. The existing codes are just OpenMVS as I said in the README. I have not released the code for transforming the  generated pfm files to the format of dmp. You can  refer to  https://github.com/cdcseacave/openMVS/blob/master/libs/MVS/Interface.h#L631

**Comment by Bill-WangJiLong** [comment]
> > pfm pfm文件文件个设法设法设法生成了运行但是但是但是但是运行运行但是后后后后后后，pfm pfm pfm pfm文件文件文件文件文件文件文件文件根本根本根本没有没有被被被被采用采用采用。依赖于依赖于依赖于依赖于依赖于依赖于依赖于依赖于依赖于还是还是还是还是pfm文件无关？
> > 请确认这一点。谢谢！
> 
> 是的你是对的。正如我在README中所说的，现在有的代码只是OpenMVS。我没有发布将生成的pfm文件转换为dmp格式的代码。可以参考 [https://github.com/cdcseacave/openMVS/ blob/master/libs/MVS/Interface.h#L631](https://github.com/cdcseacave/openMVS/blob/master/libs/MVS/Interface.h#L631)


Have you solved this problem？

**Comment by chaytonmin** [comment]
> > > pfm pfm文件文件个设法设法设法生成了运行但是但是但是但是运行运行但是后后后后后后，pfm pfm pfm pfm文件文件文件文件文件文件文件文件根本根本根本没有没有被被被被采用采用采用。依赖于依赖于依赖于依赖于依赖于依赖于依赖于依赖于依赖于还是还是还是还是pfm文件无关？
> > > 请确认这一点。谢谢！
> > 
> > 
> > 是的你是对的。正如我在README中所说的，现在有的代码只是OpenMVS。我没有发布将生成的pfm文件转换为dmp格式的代码。可以参考 [https://github.com/cdcseacave/openMVS/ blob/master/libs/MVS/Interface.h#L631](https://github.com/cdcseacave/openMVS/blob/master/libs/MVS/Interface.h#L631)
> 
> Have you solved this problem？

The code for converting the output of MVSNet to the DMap format of OpenMVS has been open-sourced :https://github.com/chaytonmin/DeepMVS/blob/master/depth2dmap.py

**Comment by zongh5a** [comment]
It seems that DensifyPointCloud.exe does not have an option to choose a local dmap, and it recalculates the dmap every time it runs. Is there any solution to this?
